external/tomee/README.md

```diff
@@ -0,0 +1,23 @@
+# External TomEE Tests
+
+Apache TomEE is a lightweight, yet powerful, JavaEE Application server with feature rich tooling.Moreover, its is a type of third party container tests help verify that the AdoptOpenJDK binaries are *good* by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries.  For each application, we choose to run a selection of their functional tests.  
```

Comment:
We will focus on Apache TomEE tests. So please change the first part of the sentence to be:
Apache TomEE tests are part of the external third-party application tests that help verify that the Adoptium binaries are *good*

Reply:
Addressed all the changes